src/components/Link.tsx

```diff
@@ -0,0 +1,25 @@
+import NextLink from 'next/link';
+import { ReactNode } from 'react';
+import { css } from '@emotion/react';
+
+type LinkProps = {
+  href: string;
+  children: ReactNode;
+};
+
+const Link = ({ href, children }: LinkProps) => {
+  const linkStyle = css({
```

Comment:
Since this is not referencing any code inside the component, it may be clearer (and marginally more performant) to lift this outside of the component